### PR TITLE
Layout docs

### PR DESCRIPTION
**Pull request by tgberkeley**

### CODE REVIEW

**Review by masenf — COMMENTED** [review]
i think the ordering should be more like

- box
- stack
- flex
- grid
- container

And @picklelo wants to just get rid of `rx.section`

File: docs/styling/layout.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    ),
+    rx.box(
+        "Radix Color",
+        background_color="var(--tomato-3)",
+        border_radius="5px",
+        width="80%",
+        margin="12px",
+        padding="12px",
+    ),
+    text_align="center",
```
[the remark this answers is not in the document]
Comment (tgberkeley) [reply docs/styling/layout.md:86]: added

**Comment by tgberkeley** [comment]
removed rx.section

File: docs/styling/layout.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+`rx.box` is a generic component that can apply any CSS style to its children. It's a building block that can be used to apply a specific layout or style property.
+
+**When to use:** Use `rx.box` when you need to apply specific styles or constraints to a part of your interface.
+
+
+```python demo
+rx.box(
+    rx.box(
+        "CSS color",
+        background_color="yellow",
```
Comment (Alek99) [inline docs/styling/layout.md:20]: Should use the color module for all the colors here rx.color or use the color scheme prop.

See the theming docs you don't need to reference the color modules in a string like this, can use rx.color https://reflex.dev/docs/styling/theming/

**Review by Alek99 — CHANGES_REQUESTED** [review]

**Review by picklelo — APPROVED** [review]

File: docs/styling/layout.md (hunk 2 above)
Comment (tgberkeley) [reply docs/styling/layout.md]: changed

**Review by masenf — APPROVED** [review]

**Pull request merged**